sanic/sanic.py

```diff
@@ -296,18 +295,11 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
         :param sock: Socket for the server to accept connections from
         :param workers: Number of processes
                         received before it is respected
-        :param loop: asyncio compatible event loop
         :param protocol: Subclass of asyncio protocol class
         :return: Nothing
         """
         self.error_handler.debug = debug
         self.debug = debug
-        if loop is not None:
```

Comment:
I think keeping them in for `0.3.0` and then taking them off for the next big version change will be the better choice here. Keep the messages but not the functionality of the loop is what I meant.